pkg/rancher-desktop/components/TheTitle.vue

```diff
@@ -31,6 +28,23 @@ export default Vue.extend({
         this.isChild = current.path.lastIndexOf('/') > 0;
       },
     },
+    action: {
+      handler(current) {
+        if (!current) {
+          return;
+        }
```

Comment:
This means it doesn't _remove_ the action; is that correct? (But I see that switching away from the page does work, so I must be missing something…)

Reply:
The watcher will trigger each time `action` changes, which includes transitions from `ImagesButtonAdd` => `null`/`undefined`/`''`. So this just means that we don't want to attempt to import/register a component after it gets flushed.

Reply:
But that means we also never assign `this.dynamicComponent` in this case; yet the button does get removed…

Reply:
The presence of a value for `action` is what drives the rendering of the components so it doesn't really matter what the value of `dynamicComponent` is. I'm open to explicitly assigning `dynamicComponent` to `null` and updating the template so that it conditionally renders based on `dynamicComponent`. It might technically be more correct this way..